root/components/Table.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {useTable} from 'react-table';
+
+import loopParity from '../utility/loopParity';
+
+const Table = (({columns, data}: any) => {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({
+    columns,
+    data,
+  });
+
+  /* eslint-disable react/jsx-key */
+  return (
+    <table {...getTableProps({className: 'tbl'})}>
+      <thead>
+        {headerGroups.map(headerGroup => (
+          <tr {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map(column => (
```

Comment:
Can we move some of these functions that don't depend on any variables from the closure to the module scope, here and below? That way the component can reuse the same function objects on every render.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1246/commits/6ee5d3d424b1dda8aecefe6be1b9f7d2ac4092e1#diff-dfa9d33d3b7fc4efebab124f77237409R15-R64